webpack.config.js

```diff
@@ -12,10 +13,21 @@ module.exports = {
     },
     module: {
         loaders: [
-            { test: /\.css$/,  loader: 'style!css' },
+            {
+                test:   /\.css$/,
+                loader: "style!css!postcss-loader"
+            },
             { test: /\.html$/, loader: 'html' },
         ]
     },
+    postcss: function () {
+        return [
+          prefix({
+            prefix: '.SGMAPembauche ', // <--- notice the trailing space!
+            exclude: ['.SGMAPembauche'] // in order to style the prefix itself in  the .css
```

Comment:
Not very important, but duplicate space in comment + “in order to” redundant as it's an inline comment, so it necessarily explains why the code to the left is here.

Slightly more important: it is not the _prefix_ that is styled, but the _container_.